pkg/datapath/linux/node.go

```diff
@@ -1213,7 +1212,7 @@ func (n *linuxNodeHandler) nodeUpdate(oldNode, newNode *nodeTypes.Node, firstAdd
 			n.enableSubnetIPsec(n.nodeConfig.IPv4PodSubnets, n.nodeConfig.IPv6PodSubnets)
 		}
 		if firstAddition && n.nodeConfig.EnableIPSec {
-			n.registerIpsecMetricOnce()
+			n.enableIpsecMetric()
```

Comment:
I wonder if we would want to start emitting this metric right away, so that we know what the xfrm counts are prior to any ipsec reconciliation (i.e. for potentially leftover state between restarts?).